article/experiment.tex

```diff
@@ -1,3 +1,31 @@
 \section{Experiment}
-
 \input{article/equal-data.tex}
+
+\section{Simple price aware extension of LightGCN}
+To investigate the extension of LightGCN, we investigate the performance of Price-Aware Recommendation, LightGCN, GCN, NGCF, GCMC, adapted LightGCN with Price and category nodes in the input data and NGCF with a price and category extension.
+The hyperparameter $X$ is tested with the values of $0.0$, $0.5$, $1.0$ and $2.0$ to see the effect of this as input data has on the final result.
```

Comment:
beskriver vi tidligere hvad X gør? aka en vægt for cat og pris

Reply:
Vi skriver "If there is a connection in $R_{ic}$ or $R_{ip}$ this value is a hyperparameter $X$ with a value $x>0$, otherwise it is 0." i implementationsafsnittet. Ved ikke om det er nok.